### ISSUE TITLE
Portable text fields don't update when navigating to other entries (via search)

### ISSUE THREAD

**Issue by robinpyon** [sanity-io/sanity#1584]
**Describe the bug**

Portable text fields don't update when navigating to other entries via search.

This only occurs when:
- you're on an entry's 'edit' URL, e.g. `/desk/edit/:entry_type/:entry_id` (as opposed to a URL when navigating through panes, e.g. `/desk/:entry_type;:entry_id`)
- you're going to another entry of the same type

**To Reproduce**

1. Go to your studio's home page by clicking the logo in the top left
2. Search and select an entry with portable text
(Alternatively, directly visit an 'edit' entry URL)
3. Search for another entry with the same type and select it

**Expected behavior**

The portable text field updates to reflect the new entry

**Screenshots**

![Kapture 2019-10-31 at 18 36 27](https://user-images.githubusercontent.com/209129/67976285-b3090d00-fc0d-11e9-88d5-74e18d01b6e0.gif)

**Which versions of Sanity are you using?**

```
@sanity/cli              0.142.2 (latest: 0.144.0)
@sanity/base             0.144.1 (up to date)
@sanity/components       0.144.2 (up to date)
@sanity/core             0.144.0 (up to date)
@sanity/default-layout   0.144.2 (up to date)
@sanity/default-login    0.144.0 (up to date)
@sanity/desk-tool        0.144.2 (up to date)
@sanity/rich-date-input  0.144.0 (up to date)
```

**What operating system are you using?**

`OS X 10.15`

**Which versions of Node.js / npm are you running?**

```
v10.16.0
6.9.0
```

**Additional context**

Navigating to an entry of another content type leads you to an intermediate page which redirects and as a result, shows the correct content.

Interestingly (and fortunately!) enough, edits to the portable text field aren't respected when this happens - which is probably a good thing as it'd lead to accidentally blowing away content! 

Example below:

![Kapture 2019-10-31 at 18 37 47](https://user-images.githubusercontent.com/209129/67976289-b603fd80-fc0d-11e9-9b68-ceea1bee025e.gif)

**Comment by skogsmaskin** [comment]
Thank you for reporting @robinpyon, we will look at this shortly!

**Comment by rexxars** [comment]
This was fixed in 0.144.3, thanks for reporting!
